stellarphot/settings/models.py

```diff
@@ -52,6 +54,21 @@
 )
 
 
+def _extract_short_description(docstring: str) -> str:
```

Comment:
Clever way to keep up with the docstring changes. :)

Reply:
Yeah, also avoids `ipyautoui` using _the entire docstring_ as the brief description...